### ISSUE TITLE
(Suggestion) add config option for repl timeout time

### ISSUE THREAD

**Issue by boogie666** [bhauman/lein-figwheel#605]
Hi,

I've noticed that when using spec.alpha and test.check the repl times out a bit to fast.

as far as i can tell this line is the problem
https://github.com/bhauman/lein-figwheel/blob/master/sidecar/src/figwheel_sidecar/repl.clj#L33

Would it be possible to pull the 8000ms as a config? and leave 8000ms as the default?

I think 8 seconds is to slow for test.check to generate 100 samples of a more complicated spec.
Basically any sample size larger then 30 takes more then 8 seconds on my machine. 

P.S. I'd be happy to make the change my self and submit a PR. just need some time to figure out how to start 😜

**Comment by bhauman** [comment]
Yes this sounds good :) Go for it!

**Comment by boogie666** [comment]
i've managed to add a timeout option.

Quick question before i submit a PR.  is the name and the docu text ok? or should i change it to something else?

See here:
https://github.com/boogie666/lein-figwheel/blob/master/sidecar/src/figwheel_sidecar/schemas/config.clj#L209

**Comment by bhauman** [comment]
Perhaps `repl-eval-timeout`?

**Comment by boogie666** [comment]
ok, will change that, 

cheers
